CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 All notable changes to the "vscode-iot-workbench" extension will be documented in this file.
 
+## Version 0.10.16
+
+- Release data: October 28, 2019
+
+### Fixed
+
+- Resolve compatibility issue with VS Code 1.39.
+- Add c_cpp_properties.json for iot workbench example project.
+
+### Added
+
+- Support build with Vcpkg for PnP generated CMake project.
+- Include source code of Azure IoT Device C SDK as sub-directory of the generated project.
+- Update PnP CodeGen flow to allow configuring project template and IoT Device SDK reference type.
```

Comment:
Optimize Plug and Play CodeGen flow, now developer can select use Azure IoT device C SDK from source code or vcpkg.